### PR TITLE
Adding myself. Without fediverse accounts.

### PR DESCRIPTION
**Pull request by Toasterson**
Apart from adding myself Kornel asked people to add their Mastodon but I didn't find a field for that yet.

https://mastodon.social/@kornel/109347896511207904

### CODE REVIEW

File: data/toasterson.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "Till Wegmueller",
+    "email": "till.wegmueller@openflowlabs.com",
+    "website": "https://www.openflowlabs.ch",
+    "blog": "https://www.wegmueller.it",
+    "irc": "toasterson",
+    "irc_channels": [],
+    "discourse": "toasterson",
+    "reddit": "toasterson",
+    "twitter": "@TheRealToaster",
+    "mastodon": "@Toasterson@chaos.social"
```
Comment (nrc) [inline data/toasterson.json:11]: JSON error: missing comma

File: data/toasterson.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (nrc) [inline data/toasterson.json:1]: The filename needs to match your username, including case

Comment (Toasterson) [reply data/toasterson.json]: Oh, fixed.

**Comment by LeoniePhiline** [comment]
@nrc Is `"mastodon"` already going to be supported, or shall I open an issue / PR?

**Pull request merged**